tools/labs/qemu/kernel_config.x86

```diff
@@ -74,6 +74,26 @@ CONFIG_DEBUG_SLAB=y
 CONFIG_DEBUG_KMEMLEAK=y
 CONFIG_PROVE_LOCKING=y
 CONFIG_DEBUG_ATOMIC_SLEEP=y
-# CONFIG_FTRACE is not set
 # CONFIG_X86_VERBOSE_BOOTUP is not set
 # CONFIG_X86_DEBUG_FPU is not set
+
+
+# for perf_events:
+CONFIG_PERF_EVENTS=y
+# for stack traces:
+CONFIG_FRAME_POINTER=y
```

Comment:
Did you use ```make savedefconfig``` ?
```
~/kernel $ ARCH=i386 make menuconfig
~./kernel $ # set config options
~/kernel $ ARCH=i386 make savedefconfig
# file saved in defconfig is the new default config that we need to copy over kerne_config.x86
```

Reply:
What for?

Reply:
Because editing the config file by hand sometimes can result in unexpected results. Also, `make savedefconfig` removes from the actual defconfig  the entries that otherwise would be selected by default keeping the defconfig minimal.

Reply:
I did a clean beforehand, so I didn't have any issue.